OpenRA.Mods.Common/Traits/World/LobbyOptionCategory.cs

```diff
@@ -0,0 +1,31 @@
+#region Copyright & License Information
+/*
+ * Copyright 2007-2020 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation, either version 3 of
+ * the License, or (at your option) any later version. For more
+ * information, see COPYING.
+ */
+#endregion
+
+using OpenRA.Traits;
+
+namespace OpenRA.Mods.Common.Traits
+{
+    [TraitLocation(SystemActors.World | SystemActors.World)]
+    [Desc("Allows to group MapOptions in categories.")]
+    public class LobbyOptionCategoryInfo : TraitInfo<LobbyOptionCategory>
+    {
+        [Desc("Category id.")]
+        public readonly string Category = "";
```

Comment:
IMO it would be cleaner to define these as `[FieldLoader.Require]` and default them to null?